### PR TITLE
[AT][Search][form] testSearchLanguageByName_HappyPath() <verafes>

### PR DESCRIPTION
**Pull request by verafes**
testSearchLanguageByName_HappyPath added
[US] - https://trello.com/c/GRtgfi9k/45-usfunctionalsearch-search-for-language-field 
[TC] - https://trello.com/c/7Op5mHP4/132-tcsearchform-verify-if-user-is-searching-for-programming-language-by-name-he-can-see-the-list-of-relative-results-verafes
[AT] - https://trello.com/c/Ld6ePLq6/133-atsearchform-testsearchforlanguagebynamehappypath-verafes

### CODE REVIEW

File: src/test/java/VerafesTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+        WebElement searchForField = getDriver().findElement(By.name("search"));
+        searchForField.click();
+        searchForField.sendKeys(LANGUAGE_NAME);
+
+        WebElement goButton = getDriver().findElement(By.name("submitsearch"));
+        goButton.click();
+
+        List<WebElement> languagesNamesList = getDriver().findElements(By.xpath("//table[@id='category']/tbody/tr/td[1]/a"));
+
```
Comment (IrynaDanilevska) [inline src/test/java/VerafesTest.java:31]: remove additional row

**Review by IrynaDanilevska — APPROVED** [review]

**Review by omalanova — APPROVED** [review]
Good 👍

**Review by omalanova — APPROVED** [review]

File: src/test/java/VerafesTest.java (hunk 1 above)
Comment (verafes) [reply src/test/java/VerafesTest.java:31]: please see changes in next commit

**Comment by verafes** [comment]
> Good 👍

still need approval for this PR

**Comment by verafes** [comment]
@omalanova, Thank you for commenting changes, but PR still needs to be approved

**Review by OlgaPrice — APPROVED** [review]

**Pull request merged**